templates/app-service-linux.json

```diff
@@ -63,31 +70,31 @@
         "siteConfig": {
           "alwaysOn": true,
           "appSettings": "[parameters('appServiceAppSettings')]",
+          "connectionStrings": "[parameters('appServiceConnectionStrings')]",
+          "linuxFxVersion": "[if(not(parameters('deploymentUsingSlots')), parameters('runtimeStack'), json('null'))]"
```

Comment:
Does setting this value to 'null' not break the currently running application?

Reply:
That's a valid point! I think instead of null I need to reference existing resource property value

Reply:
I can confirm that json('null') doesn't break the currently running application or overwrite existing property value. It is not same as null or "", it is equivalent to else do nothing. I have tested it few times successfully.

Reply:
Perfect